src/render/SDL_render.c

```diff
@@ -967,6 +1001,14 @@ SDL_Renderer *SDL_CreateRenderer(SDL_Window *window, int index, Uint32 flags)
     Android_ActivityMutex_Lock_Running();
 #endif
 
+/*CPU acceleration runs faster than integrated graphichs on Loongson at pressent.*/
+#if defined(__linux__) && defined(__loongarch__)
+    if (SDL_CheckIntegratedGraphics()) {
+        SDL_SetHint(SDL_HINT_FRAMEBUFFER_ACCELERATION, "0");
```

Comment:
I'm not sure it's appropriate to override hints from the user program. Should this be a hint returning